engine/src/test/java/io/zeebe/engine/processing/streamprocessor/ReprocessingIssueDetectionMultiplePartitionTest.java

```diff
@@ -83,8 +83,7 @@ public void setup() {
   @Test
   public void shouldIgnoreDeploymentDistribution() {
     // given
-    // force that a deployment DISTRIBUTED event is written on reprocessing which was not written on
-    // the log stream before
+    // the deployment can be distributed on reprocessing
```

Comment:
Not sure whether this test makes still sense? But I thought maybe we can have a different test where we verify that we not call distribute on reprocessing. I have new logic where I can await reprocessing, which means we know whether before distribute was called. What do you think?

Reply:
I think the test still makes sense for the detection. However, I can add another that `DeploymentDistributor` is not called on reprocessing. Please point me to the example :)

Reply:
https://github.com/zeebe-io/zeebe/pull/5761/files#diff-b31c3c84b754243e651147040006e4acd695b0315a1f9832f63b6e20b0a3f7f3R199-R204

Reply:
I think we wait until this PR is merged than you can rebase

Reply:
Ok Chris :sweat_smile: I rewrote the test to verify that the deployment distributor is not invoked on reprocessing. Please have a look :)